lore/settings.py

```diff
@@ -348,3 +348,4 @@ def get_var(name, default):
     }
 }
 HAYSTACK_SIGNAL_PROCESSOR = 'haystack.signals.RealtimeSignalProcessor'
+XANALYTICS_URL = get_var('XANALYTICS_URL', "")
```

Comment:
Have we pinned down the auth scheme at all?  This will be fine if we end up using basic auth, but if we need token auth, we are going to need a second var to store the security token in it.